docs/src/app/versions.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Yeah, like I said, it's a nice-to-have. I'll kill it for now. We have more important things to worry about!